src/routes/movies.py

```
@@ -370,117 +781,202 @@
+    result = await db.execute(
+        select(StarModel)
+        .offset(skip)
+        .limit(limit)
+    )
+    return result.scalars().all()
+
+
+@router.get("/{star_id}", tags=["Stars"], response_model=StarDetailSchema)
+async def get_star_detail(
+        star_id: int,
+        db: AsyncSession = Depends(get_db)
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)